wandb/plots/heatmap.py

```diff
@@ -0,0 +1,60 @@
+import wandb
+from wandb import util
+from wandb.plots.utils import test_missing, test_types, encode_labels
+np = util.get_module("numpy", required="Logging plots requires numpy")
+scikit = util.get_module("sklearn")
```

Comment:
We should put these in the heatmap method itself.   Putting a required get_module here will make it required for the whole library because we import `wandb.plots` in the `wandb` module.

Reply:
fixed [in 909!](https://github.com/wandb/client/pull/909/commits/d7aa158d5ee1efa819691dbe0f1c5a8a53f7e152)